### PR TITLE
Fix for Accessibility issues with Tables (highlighted by UCOP electro…

### PR DESCRIPTION
**Pull request by johnpinto1**
…nic accessibility review)

Changes:
  - Added scope="col" to table header tags <th> that were missing them.
  - The Jquery dropdown menus in tables are closed when the table cell
containing it is out of focus.

(The issue the fontawesome sort icons are read as 'column # link'.
Add some screen reader only text. This issue was sorted by someone else
who added ".screen-reader-text" class to the sort_link_name() method in
app/controllers/concerns/paginable.rb.)

Fix for issue #1455.

I believe I fixed the issue causing the previous PR (that I closed #2239 yesterday)

### CODE REVIEW

**Comment by johnpinto1** [comment]
@xsrus @briri Travis failing on
```
Vulnerabilities found!

The command "bundle exec bundle-audit check --update --ignore CVE-2015-9284" exited with 1.
```

File: app/javascript/utils/outOfFocus.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+$(() => {
+  // This bit of code currently only closes dropdown menus in a table when the focus is no longer in the containing cell.
```
Comment (briri) [inline app/javascript/utils/outOfFocus.js]: Thanks for adding all of this commentary @johnpinto1 It will be helpful in the future!

**Comment by briri** [comment]
Thanks @johnpinto1. Have you tried running `bundle update` to see if that rubzip can be updated to 1.3+? https://nvd.nist.gov/vuln/detail/CVE-2019-16892

Its used by the htmltoword gem but I haven't dug into how its being used. We don't zip files (that I know of) so it may be an ignorable issue. Its better to upgrade though if we can imo

**Comment by xsrust** [comment]
Looks like a `bundle update rubyzip`, and then committing `Gemfile.lock` should  fix the problem.

It looks like it's been a while since [htmltoword](https://github.com/karnov/htmltoword) got an update

**Comment by johnpinto1** [comment]
@xsrust  @briri Did a bundle update as suggested. Just need to wait for Travis report.

**Comment by johnpinto1** [comment]
@xsrust @briri The `bundle update` did not fix Travis.

**Comment by xsrust** [comment]
We got a new error for Travis, this  time complaining  about our cookie serialization.
I created #2255 to track the problem and will submit  a PR to development to update the insecure setting.
In the  meantime, I've re-started the remaining tests on the PR so it goes through the rails/js checks  against your  code

**Comment by xsrust** [comment]
This is passing now that Development is patched.

**Pull request merged**